packages/babel-plugin/src/styled/__tests__/index.test.tsx

```diff
@@ -176,7 +176,7 @@ describe('styled component transformer', () => {
       \`;
     `);
 
-    expect(actual).toInclude("import React from 'react';");
+    expect(actual).toInclude("import React from'react';");
```

Comment:
Is the lack of space here after `from` correct? (is that actually valid JavaScript?). Seems to be some of that above as well on line 143.

Reply:
It's valid. It's just the test code compressing everything so we have consistent white space